### ISSUE TITLE
Early stopping in multi-task learning

### ISSUE THREAD

**Issue by Yingyingzhang15** [fchollet/keras#3699]
Hi. I'm trying to implement [Facial landmark detection by deep multi-task learning](http://www.eecs.qmul.ac.uk/~ccloy/files/eccv_2014_deepfacealign.pdf)(ECCV2014)
using Keras.In this paper, different tasks are stopped seperately. Is there any early stopping function used in multi-task learning in Keras？　If not, how can I write my own code to stop the training of specific task ?
Thank you ！
@farizrahman4u

**Comment by carlthome** [comment]
What's a "task"? A separate objective that's minimized independently?

**Comment by Yingyingzhang15** [comment]
@carlthome Yes, you can think about a muti-output model

**Comment by carlthome** [comment]
Looking at your link, I think your answer should be no. They minimize a single objective of weighted terms where a particular term is viewed as some kind of task. No?

**Comment by Yingyingzhang15** [comment]
@carlthome You mean Keras cannot early-stop a specific task?Do you have any ideas about how o write the codes by myself?

**Comment by carlthome** [comment]
Early stopping per output could be performed with a model callback that freezes weights or an output's loss when a specific validation loss starts to increase too much.
